cpu/esp32/include/gpio_arch.h

```diff
@@ -54,6 +53,19 @@ typedef enum
     _NOT_EXIST  /**< pin cannot be used at all */
 } gpio_pin_usage_t;
 
+#endif /* DOXYGEN */
+
+/**
+ * @brief   Current an output pin can drive in active and sleep modes
```

Comment:
Could you somehow rewrite this sentence?

Reply:
Hm, it says what it means. Any suggestion?

Reply:
Maybe just "The current an output pin can drive in active and sleep modes"